worksheet_A/TerminalHacking.cpp

```diff
@@ -34,13 +44,71 @@ int main()
 	}
 
 	// Display all words
-	for each (std::string word in options)
-	{
-		std::cout << word << std::endl;
+	int i = 0;
+	int k = 0;
+
+	for each (std::string word in options){
+		std::cout << "\t" <<word << std::endl; // no matches but print it
 	}
 
 	// TODO: implement the rest of the game
+	// get user input of their chosen word
+
 
-    return 0;
+	int x = 4; //to limit 4 guesses
+	std::string Guess; //variable to hold user input
+	std::string request; //petty text variable
+	while (x > 0) { //while we have any guesses left
+		Guess = ""; //reset varialbe to empty
+		if (x == 1) { //
+			request = "guess";
+		}
+		else {
+			request = "guesses";
+		}
+		std::cout << "Enter your guess. You have " << x << " " << request << " remaining." << std::endl;
+		std::getline(std::cin, Guess, '\n');
+		
+			std::transform(Guess.begin(), Guess.end(), Guess.begin(), ::toupper);
+			if (options.find(Guess) != options.end()) {
```

Comment:
This and the previous line would benefit from comments, i.e. `// Convert the guessed word to upper case` and `// Is the guessed word in the list of options?`